sdk/mongocluster/Azure.ResourceManager.MongoCluster/samples/Generated/Samples/Sample_MongoClusterPrivateEndpointConnectionResource.cs

```diff
@@ -21,7 +21,7 @@ public partial class Sample_MongoClusterPrivateEndpointConnectionResource
         public async Task Get_GetAPrivateEndpointConnectionOnAMongoClusterResource()
         {
             // Generated from example definition: 2024-07-01/MongoClusters_PrivateEndpointConnectionGet.json
-            // this example is just showing the usage of "PrivateEndpointConnectionResource_Get" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "PrivateEndpointConnections_get" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it should be `PrivateEndpointConnections_get`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/mongocluster/DocumentDB.MongoCluster.Management/PrivateEndpointConnection.tsp#L14C11-L14C38